db/ddl.sql

```diff
@@ -27,12 +29,27 @@ CREATE TABLE IF NOT EXISTS trades (
     price_cents INTEGER NOT NULL,
     quantity INTEGER NOT NULL,
     timestamp REAL DEFAULT CURRENT_TIMESTAMP,
-    FOREIGN KEY (market_id) REFERENCES markets (id)
+    FOREIGN KEY (market_id) REFERENCES markets (id) ON DELETE CASCADE
```

Comment:
we shouldn't be deleting trades if markets get deleted, this makes PnL inaccurate

Reply:
@alicejjliu You really shouldn't be deleting a market, my reasoning is that this is the equivalent of deleting a spreadsheet

But I think there should be an option to archive so it doesn't show up in /home